src/components/personal/UserInfo.vue

```diff
@@ -5,12 +5,12 @@
         <div class="card user-info__row__block__card">
           <div class="user-info__row__block__card__card-image">
             <img class="user-info__row__block__card__card-image__image" src="../../assets/forest.jpg" height="532"
```

Comment:
You don't need to construct such class names. e.g:
No need to put additional `user-info__row` if there's already `row`.
Every division by `__` means division by blocks. If block is independent you can omit it's parent classname. For example: 
`user-info` is parent of everything, so you could make just `user-info__block` and `user-info__card`.
More then one `__` is very rare case. If you see there need to make it more then once - you can omit the furthest parent in class name. Like `user-info__card__card-image` -> `card__card-image`. Or `card__card-image__image` -> `card-image__image`.

Reply:
The algorythm is easy - if new block (div or anything) can leave without it's parent OR can be used in other parent -> it's independent block